setup.py

```diff
@@ -64,20 +63,21 @@ def _unavailable(self, exc):
         print(exc)
 
 
-if Feature:
-    # Optional C extension module for speeding up Genshi:
-    # Not activated by default on:
-    # - PyPy (where it harms performance)
-    # - CPython >= 3.3 (the new Unicode C API is not supported yet)
-    speedups = Feature(
-        "optional C speed-enhancements",
-        standard = not is_pypy,
-        ext_modules = [
-            Extension('genshi._speedups', ['genshi/_speedups.c']),
-        ],
-    )
-else:
-    speedups = None
+# Optional C extension module for speeding up Genshi
+speedups = Extension('genshi._speedups', ['genshi/_speedups.c'])
+ext_modules = []
+# Not activated by default on:
+# - PyPy (where it harms performance)
+try:
+    _speedup_enable_default = 0 if is_pypy else 1
+    _speedup_enabled = int(os.getenv('GENSHI_BUILD_SPEEDUP', _speedup_enable_default))
+except ValueError:
+    import warnings
+    warnings.warn('GENSHI_BUILD_SPEEDUP was defined to something other than 0 or 1; defaulting to not build...')
+    _speedup_enabled = False
+
+if _speedup_enabled:
+    ext_modules = [speedups]
```

Comment:
Could we clean this up slightly by doing here:
```python
ext_modules = []
if _speedup_enabled:
    ext_modules.append(Extension('genshi._speedups', ['genshi/_speedups.c']))
```

Reply:
Defining an Extension() does not enable it, and always parsing it can catch misuse even when it's disabled (which is the kind of thing that could have caught the 'Feature' removal). But I guess it's not a huge issue either way.